.github/workflows/test-pool.yml

```diff
@@ -60,7 +58,7 @@ jobs:
       run: |
         python tests/test_server_pool.py
 
-    - uses: EndBug/add-and-commit@v7.0.0
+    - uses: EndBug/add-and-commit@v9.1.3
```

Comment:
Let's use this opportunity to use the commit hash per actions best practices (and indicate the tag in a comment)